tools/update-codegen.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o nounset
+#set -o pipefail
```

Comment:
Should this be left commented? Or is it an omission? Or should we just remove it?

Reply:
it's not applicable for alpine (the base image), let me remove it then :)